modules/Story/Story.tsx

```diff
@@ -32,10 +38,13 @@ export async function Story({ story }: Props) {
         <article className={styles.story}>
             <div className={styles.container}>
                 <Embargo story={story} />
+                {withHeaderImage === 'above' && headerImageDocument && (
+                    <HeaderImageRenderer nodes={headerImageDocument} />
+                )}
```

Comment:
I wonder if this cannot become a part of the `<HeaderRenderer>`. Sounds about right. Not insisting though :)